docs/modules/migrate/pages/lts.adoc

```diff
@@ -0,0 +1,128 @@
+
+|Choose if you value stability over new feature adoption, you want to upgrade less frequently, and you want full or extended support
+|Choose if you value new features over stability, you are willing to upgrade several times a year, and you are happy with maintenance support for the life of the STS
+|===
+
+== Hazelcast 5.5 LTS
+
+The Hazelcast 5.5 LTS release supports direct upgrade as follows:
+
+[cols="1,1"]
+|===
+|Upgrade From|LTS Release Support
+
+|5.4
+|All features
+
+|5.2
+
+5.3
+
+a|* IMap data structure. This includes the API, configuration options, and persistence
+* CP subsystem. This inclues the API and peristence
+* WAN replication. Ongoing replication is supported during the upgrade. This includes both source and target; however, the source and the target cannot be upgraded simultaneously
+|===
+
+== Mixed-version Clusters
+
+Your LTS release member can join clusters that include members from an earlier supported version. You cannot have members from more than two different minor versions in the same cluster.
+
+The following table provides an example of the behaviour:
+
+[cols="1,1,1,1"]
+|===
+|Action|Member Version|Cluster Version|Description
+
+|Form cluster
+|5.2
+|5.2
+|5.2 cluster forms
+
+|Member tries to join cluster
+|5.3
+|5.2
+|5.3 member joins 5.2 cluster
+
+|Further member tries to join cluster
+|LTS
+|5.2
+|LTS member identifies that two different minor versions are already in the cluster and rejects joining the cluster
+|===
+
+The scenario outlined below can occur because the logic for a mixed-version cluster is introduced in the first LTS release and cannot be enforced by earlier releases.
+However, the scenario is not supported and results from such a cluster would not be reliable. 
+
+Hazelcast recommends that you avoid the following scenario:
+
+[cols="1,1,1,1"]
+|===
+|Action|Member Version|Cluster Version|Description
+
+|Form cluster
+|5.2
+|5.2
+|5.2 cluster forms
+
+|Member tries to join cluster
+|LTS
+|5.2
+|Member identifies that only one minor version is already in the cluster and joins the cluster
+
+|Further member tries to join cluster
+|5.3
+|5.2
+|Member joins the cluster
+|===
+
+== High-level Process
+
+The process for upgrading to an LTS release is the same as for a rolling upgrade. To summarize, the process involves the following steps:
+
+. For each member in the cluster:
+
+.. Shut down the member and wait for all partitions to migrate to the rest of the cluster
+.. Upgrade the member's codebase
+.. Start the member and wait for it to join the cluster
+
+. Trigger the rolling upgrade on the cluster
+
+For further information on performing a rolling upgrade, including the steps to take, compatibility guarantees and any constraints, see xref:maintain-cluster:rolling-upgrades.adoc[]. 
+
+=== Restart Members on Previous Version
+
+If you experience any issues during the rolling upgrade, you can recover to the previous version by restarting the members.
```

Comment:
This can be done only if the version hasn't been upgraded (either automaticaly or via API/script).